src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -791,6 +793,7 @@ def enable_zip_deploy(cmd, resource_group_name, name, src, timeout=None, slot=No
     from azure.cli.core.util import should_disable_connection_verify
     # check if the app is a linux web app
     app_is_linux_webapp = is_linux_webapp(app)
+    app_is_function_app = is_functionapp(app)
```

Comment:
Can we please add one liner on the def of "enable_zip_deploy" function to make it more readable.